src/main/kotlin/com/github/oxc/project/oxcintellijplugin/Constants.kt

```diff
@@ -2,6 +2,7 @@ package com.github.oxc.project.oxcintellijplugin
 
 object Constants {
 
-    val CONFIG_FILES = listOf("oxlintrc.json", "oxlint.json", ".oxlintrc.json", ".oxlint.json",
-        ".oxlintrc")
+    val ESLINT_CONFIG_FILES = listOf(".eslintrc", ".eslintrc.json")
```

Comment:
I think these are the only config files that Oxlint targets currently, but it can be updated to whatever is necessary.